testcontainers/src/core.rs

```diff
@@ -14,4 +14,4 @@ pub(crate) mod logs;
 pub(crate) mod macros;
 pub(crate) mod mounts;
 pub(crate) mod network;
-pub(crate) mod ports;
+pub mod ports;
```

Comment:
Do we really need to make it public? I guess it's enough to re-export one particular type (`ExposedPort`)

Other types were hidden before and don't think we need them now 🤔

P.S: in general, we probably need to reorganize current state of imports (make some `mod`s public, remove excessive reimports and introduce `prelude` module). But it's out of scope for the PR

Reply:
Done